.bumpversion.cfg

```diff
@@ -1,20 +1,20 @@
 [bumpversion]
-current_version = 0.17.3.dev0
+current_version = 0.17.3-dev0
 commit = true
 tag = true
 tag_name = {new_version}
 message = chore: bump version: {current_version} -> {new_version}
-parse = (?P<major>\d+)\.(?P<minor>\d+)(\.(?P<patch>\d+)((?P<pre>\.dev|a|b|rc|final|post)?((\.dev)?(?P<dev>\d+))?))?
+parse = (?P<major>\d+)\.(?P<minor>\d+)(\.(?P<patch>\d+)((?P<pre>-(dev|a|b|rc|final|post))?(?P<dev>\d+)?))?
 serialize = 
 	{major}.{minor}.{patch}{pre}{dev}
 	{major}.{minor}.{patch}
 
 [bumpversion:part:pre]
-optional_value = final
+optional_value = -final
```

Comment:
It doesn't show up explicitly in the output, but this is the way to tell bumpversion to go through the sequence `0.x.y-dev0` -> `0.x.y-rc0` -> `0.x.y` (when it would hit `-final`, the fact that that value is optional makes the output the version with no suffix).